azext_hanaonazure/modules_sdk/version.py

```diff
@@ -9,5 +9,5 @@
 # regenerated.
 # --------------------------------------------------------------------------
 
-VERSION = "0.9.0"
+VERSION = "0.11.0"
```

Comment:
why is it a jump from 9 to 11? Is it something regenerated on our behalf?

Reply:
This is generated on our behalf; it had been bumped to 0.10.0 before, but not updated in this CLI extension codebase.